### PR TITLE
Update configuration document

### PR DESCRIPTION
**Pull request by rantan**

### CODE REVIEW

File: doc/configuration.md [hunk 1]
```diff
@@ -4,48 +4,329 @@ This document describes how to configure tapyrus-signer.
...
+threshold = 2
+aggregated-public-key = "030acd6af9..."
+node-vss = [
+  "02472012cf49fca573ca...",
+  "02ce7edc292d7b747fab...",
+  "03831a69b8009833ab5b..."
+]
+```
+
+Before generating 99 height block proof, you should startup your tapyrus-signerd daemon.
```
Comment (Yamaguchi) [inline doc/configuration.md:289]: I think `Before generating 100 height block proof` is correct.
This member can't participate 99 generation and so any messages is not sent from other signer at 99.

File: doc/configuration.md [hunk 2]
```diff
@@ -4,48 +4,329 @@ This document describes how to configure tapyrus-signer.
 
 After setting up with [How to set up new Tapyrus Signer Network](./setup.md), We can start Tapyrus Signer Network.
 
-We use some parameters introduced in setup documentation.
+`tapyrus-signerd` has two configure files. Both can be specified the path and the name with command line arguments.
 
-- `public_key` : A public key generated by using setup Step 2 
-- `node_vss`: An array of the VSS generated by using setup Step 3
+* **signer.toml**: This is a configuration file for general purpose to pass the arguments to `tapyrus-sigenrd`.
```
[the remark this answers is not in the document]
Comment (rantan) [reply doc/configuration.md:9]: I will fix. thanks.

File: doc/configuration.md (hunk 1 above)
Comment (rantan) [reply doc/configuration.md:289]: It's true. I will fix. Thanks.

File: doc/configuration.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (rantan) [reply doc/configuration.md]: fixed at 36d611ae4e37a2f32a6441881e2835958cd2c3ba

File: doc/configuration.md (hunk 1 above)
Comment (rantan) [reply doc/configuration.md]: fixed at 437acaf62e2bce035dfb35cfd4f53da556b5d9af

**Pull request merged**